.github/workflows/update-identity.yml

```diff
@@ -77,15 +105,25 @@ jobs:
                   "type": "section",
                   "text": {
                     "type": "mrkdwn",
-                    "text": ":alarm: *Updating the identity version* on `${{ inputs.targetBranch }}` failed! :alarm:\n ${{ inputs.userSlackId != '' && format('\\cc <@{0}>', inputs.userSlackId) || '' }}"
+                    "text": ":alarm: *Updating Identity version* to ${{ inputs.identityVersion }} on `${{ inputs.targetBranch }}` failed!\n"
                   }
                 },
                 {
                   "type": "section",
                   "text": {
                     "type": "mrkdwn",
-                    "text": "Please check the related workflow execution: https://github.com/camunda/camunda/actions/runs/${{ github.run_id }}"
+                    "text": "${{ inputs.userSlackId != '' && format('<@{0}>', inputs.userSlackId) || '' }} Please check this <https://github.com/camunda/camunda/actions/runs/${{ github.run_id }}|GHA workflow run>."
```

Comment:
is `userSlackId` always a concrete Slack user, or can be a Slack group?
if both, a group needs to be references differently:`<!subteam^S01234567>`.

Reply:
Right now, it is always a user because it is deferred from the current release manager in the release process.